### PR TITLE
Reviewed sv_SE locale files

### PR DESCRIPTION
**Pull request by martinpub**
Initial commit of updating the sv_SE locale with translation work done at Lund University Library.

### CODE REVIEW

File: plugins/generic/referral/locale/sv_SE/locale.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * plugins/generic/referral/locale/en_US/locale.xml
```
Comment (mtub) [inline plugins/generic/referral/locale/sv_SE/locale.xml]: should be .../sv_SE/locale.xml

File: plugins/importexport/quickSubmit/locale/sv_SE/locale.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * plugins/importexport/quickSubmit/locale/en_US/locale.xml
```
Comment (mtub) [inline plugins/importexport/quickSubmit/locale/sv_SE/locale.xml]: path -> sv_SE

Comment (martinpub) [inline plugins/importexport/quickSubmit/locale/sv_SE/locale.xml]: Thanks, I have corrected them now.

**Comment by mtub** [comment]
Thanks, @martinpub. Your contribution is very much appreciated. Will merge now.

**Comment by asmecher** [comment]
@mtub, shouldn't this also be cherry-picked to ojs-stable-2_4_6? If so, I can do that.

**Comment by mtub** [comment]
@asmecher, feel free to cherry-pick to ojs-stable-2_4_6…

**Comment by asmecher** [comment]
Done, thanks!

**Pull request merged**